mflowgen/full_chip/construct.py

```diff
@@ -25,32 +25,33 @@ def construct():
   parameters = {
     'construct_path'    : __file__,
     'design_name'       : 'GarnetSOC_pad_frame',
-    'clock_period'      : 100.0,
+    'clock_period'      : 2.0,
```

Comment:
Seems like the constraints files don't use this param... Can we make them use it?

Reply:
I can update them to use it. The trick is to put some logic to cap the rest of the SoC at 500MHz.

Reply:
We can also add a second parameter